Samples/Sources/SampleDiningPhilosophers/DistributedDiningPhilosophers.swift

```diff
@@ -76,4 +72,14 @@ final class DistributedDiningPhilosophers {
 
         try systemA.park(atMost: duration)
     }
+
+    private func isClusterFormed(_ systems: [ClusterSystem]) async throws -> Bool {
+        for system in systems {
+            let upCount = try await system.cluster.membershipSnapshot.count(withStatus: .up)
+            if upCount != systems.count {
+                return false
+            }
+        }
+        return true
```

Comment:
Implementing this but with a continuation and condition like "`awaitMembers(count: ..., statusAtLeast: .up)`" etc is actually ticket https://github.com/apple/swift-distributed-actors/issues/948 :) 

It is a less urgent one, but FYI, maybe a TODO here?